_config.yml

```diff
@@ -94,9 +94,9 @@ just_the_docs:
       name: External links
 
 
-# Enable or disable the site search
+# Enable or disable the off-the-shelf site search
 # Supports true (default) or false
-search_enabled: true
+search_enabled: false
```

Comment:
Can we rename this flag to be more specific? It confusing to parse the logic later in `default.html` what this flag does because we have a search bar irrespective of the flag being true or false

Reply:
This is defined by the underlying application we use for the documentation: https://github.com/pmarsceill/just-the-docs. Do you feel more explanation is warranted?